src/stories/Search.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Can we also add this file is TS?

Reply:
Works like a charm @hayata-suenaga @mountiny